### PR TITLE
fix(jwt): Use isJwtFeatureEnabled the same way in all places.

### PR DESCRIPTION
**Pull request by damencho**
Fixes an issue where we were showing cc button for visitors that does not have features in the token.

### CODE REVIEW

File: react/features/base/jwt/functions.ts [hunk 1]
```diff
@@ -76,23 +76,23 @@ interface IIsJwtFeatureEnabledStatelessParams {
...
+    ifNotInFeatures
 }: IIsJwtFeatureEnabledStatelessParams) {
     if (!jwt) {
         return ifNoToken;
     }
 
-    // If `features` is undefined, act as if everything is enabled.
     if (typeof features === 'undefined') {
-        return true;
+        return ifNoToken;
```
Comment (hristoterezov) [inline react/features/base/jwt/functions.ts:95]: I'm wondering shouldn't we return `ifNotInFeatures` in this use case?

File: react/features/toolbox/functions.any.ts [hunk 2]
```diff
@@ -21,12 +21,14 @@ export function isAudioMuteButtonDisabled(state: IReduxState) {
...
  *
  * @param {IReduxState} state - The state from the Redux store.
+ * @param {boolean} isModerator - Whether local participant is moderator.
  * @param {string | undefined} jwt - The jwt token.
  * @param {ILocalParticipant} localParticipantFeatures - The features of the local participant.
  * @returns {string[]} - The disabled by jwt buttons array.
  */
 export function getJwtDisabledButtons(
         state: IReduxState,
+        isModerator: boolean,
```
Comment (hristoterezov) [inline react/features/toolbox/functions.any.ts:31]: Why not get it from the state?

File: react/features/base/jwt/functions.ts (hunk 1 above)
Comment (damencho) [reply react/features/base/jwt/functions.ts:95]: Well when there are no features, we want to respect isModerator, which we pass to ifNoToken.

File: react/features/toolbox/functions.any.ts (hunk 2 above)
Comment (damencho) [reply react/features/toolbox/functions.any.ts:31]: 👍

**Review by hristoterezov — APPROVED** [review]

**Pull request merged**